src/main/resources/static/ptmexchange/index.html

```
@@ -52,10 +52,13 @@ <h1>Datasets</h1>
         <td><i>Homo sapiens</i></td><td>Human</td><td>SUMO</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/human/sumo/">Human SUMO 2025&#8209;02</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome<a/></td><td></td><td><a href="./Files/Human_GSB_SUMO_all_prots_0924.csv.gz">  Results</a></td><td><a href="./Files/Homo_one_Prot_per_Gene_cRAB_decoy.fasta.gz"> Fasta file</a></td>
       </tr>
 	  <tr class="pc-hov">
-        <td><i>Homo sapiens</i></td><td>Human</td><td>Ubiquitination</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/human/ubiquitination/">Human Ubiquitination 2025-??</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome<a/></td><td></td><td><a href="./Files/Human_GSB_????_all_prots_????.csv.gz">  Results</a></td><td><a href="./Files/Homo_one_Prot_per_Gene_cRAB_decoy.fasta.gz"> Fasta file</a></td>
+        <td><i>Homo sapiens</i></td><td>Human</td><td>Ubiquitination</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/human/ubiquitination/">Human Ubiquitination 2025-??</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome<a/></td><td></td><td><a href="./Files/Human_GSB_Ubiq_all_prots_1106.csv.gz">  Results</a></td><td><a href="./Files/2024-04-04-decoys-contam-uniprotkb_proteome_UP000005640.fasta.gz"> Fasta file</a></td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a href="https://peptideatlas.org/builds/human/ubiquitination/"> ] on line 55.

(tag-pair)